src/app.js

```diff
@@ -180,23 +180,16 @@ export function app(app) {
     }
   }
 
-  function updateElementData(element, oldData, data, cb) {
+  function updateElementData(element, oldData, data) {
     for (var name in merge(oldData, data)) {
       var value = data[name]
       var oldValue = oldData[name]
 
-      if (
-        value !== oldValue &&
-        value !== element[name] &&
```

Comment:
I didn't take away the `value !== oldValue` that's still there. And afaik `value !== element[name]` will always be the same as the first check (just slower). So it makes sense to me to get rid of the last check. But I could be missing something?